detection_rules/main.py

```diff
@@ -149,12 +151,48 @@ def view_rule(ctx, rule_id, rule_file, api_format):
     if not rule:
         client_error('Unknown format!')
 
-    click.echo(toml_write(rule.rule_format()) if not api_format else
-               json.dumps(rule.contents, indent=2, sort_keys=True))
+    if verbose:
+        click.echo(toml_write(rule.rule_format()) if not api_format else
+                   json.dumps(rule.contents, indent=2, sort_keys=True))
 
     return rule
 
 
+@root.command('export-rules')
+@click.argument('rule-id', nargs=-1, required=False)
+@click.option('--rule-file', '-f', multiple=True, type=click.Path(dir_okay=False), help='Export specified rule files')
+@click.option('--directory', '-d', multiple=True, type=click.Path(file_okay=False),
+              help='Recursively export rules from a directory')
+@click.option('--outfile', '-o', default=get_path('exports', f'{time.strftime("%Y%m%dT%H%M%SL")}.ndjson'),
+              type=click.Path(dir_okay=False), help='Name of file for exported rules')
+@click.option('--randomize-id', '-r', is_flag=True, help='Randomize rule IDs before export')
+def export_rules(rule_id, rule_file, directory, outfile, randomize_id):
+    """Export rule(s) into an importable ndjson file."""
+    from .packaging import Package
+
+    if not (rule_id or rule_file or directory):
+        client_error('Must specify a rule_id, rule_file, or directory')
+
+    rules = [r for r in rule_loader.load_rules(verbose=False).values() if r.id in rule_id] if rule_id else []
```

Comment:
What if an ID isn't found? Or if an ID is duplicated? Errors or silently drop?

Reply:
I'll add a check for missing to raise client_error 👍